Dockerfile

```diff
@@ -16,4 +16,4 @@ COPY ./test_requirements.txt /rslearn/test_requirements.txt
 RUN pip install --no-cache-dir -r test_requirements.txt
 
 COPY ./ /rslearn
-RUN pip install --no-cache-dir  .
+RUN pip install --no-cache-dir  rslearn[extra]
```

Comment:
Won't this install from pypi rather than from the current directory?

What about `pip install --no-cache-dir .[extra]`?

Reply:
Is it needed to change this or is the change to `extra_requirements.txt` sufficient?

Reply:
removed this change